src/open3DTool/fileUtils.py

```diff
@@ -0,0 +1,41 @@
+import open3d as o3d
+from src.algorithmForNN.compressUtils import FIC, LZW
+
+
+def update_label_files(
+    point_cloud: o3d.geometry.PointCloud,
+    count_of_points: int,
+    path_to_pcd_file: str,
+    path_to_label_file: str,
+    path_to_object_file: str,  # may be add in future
+    is_append_right: bool,
```

Comment:
What this option is for?

Reply:
Saving to file using MVP modifies the original binary. New points are added to the right of the "array", I hope that in the main JS tool, the segmented points can be returned to their original positions.